### PR TITLE
glass jar fixes but its atomic

### PR DESCRIPTION
**Pull request by alsoandanswer**
title

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/glassjar.dm [hunk 1]
```diff
@@ -25,50 +25,51 @@
...
 		STOP_PROCESSING(SSprocessing, S)	// No growing inside jars
 		contains = JAR_SPIDERLING
 		update_icon()
 		return
-	if(ismob(A))
-		var/mob/L = A
-		if(L.mob_size <= MOB_SMALL)
-			user.visible_message(SPAN_NOTICE("<b>\The [user]</b> scoops \the [L] into \the [src]."), SPAN_NOTICE("You scoop \the [L] into \the [src]."))
-			L.forceMove(src)
+	if(istype(A, /mob/living/simple_animal))
```
Comment (NonQueueingMatt) [inline code/game/objects/items/glassjar.dm:34]: Why are we checking specifically for simple animals?

File: code/game/objects/items/glassjar.dm [hunk 2]
```diff
@@ -25,50 +25,51 @@
...
+		var/mob/living/simple_animal/S = A
+		if(S.mob_size <= MOB_TINY)
+			user.visible_message(SPAN_NOTICE("<b>\The [user]</b> scoops up \the [S] into [src]."), SPAN_NOTICE("You scoop up \the [S] into [src]."))
+			playsound(src, pickup_sound, PICKUP_SOUND_VOLUME)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/glassjar.dm:38]: This doesn't look like a bugfix to me.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**